<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="refresh_pushButton">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="search_pushButton">
        <property name="text">
         <string>检索</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="manage_pushButton">
        <property name="text">
         <string>管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="order_pushButton">
        <property name="text">
         <string>排序</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="view_pushButton">
        <property name="text">
         <string>视图</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>911</width>
           <height>449</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="status_text_label">
        <property name="text">
         <string>状态栏</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="last_page_pushButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>上一页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="page_num_lineEdit">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>4</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="page_num_all_label">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next_page_pushButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>下一页</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_file_action"/>
   <addaction name="open_folder_action"/>
   <addaction name="edit_tab_action"/>
   <addaction name="downlowd_infoaction_action"/>
   <addaction name="downlowd_img_action"/>
   <addaction name="open_setting_action"/>
  </widget>
  <action name="openfile">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="openfolder">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="open_setting">
   <property name="text">
    <string>打开设置</string>
   </property>
  </action>
  <action name="edit_tab">
   <property name="text">
    <string>编辑标签</string>
   </property>
  </action>
  <action name="downlowd_info">
   <property name="text">
    <string>下载信息</string>
   </property>
  </action>
  <action name="downlowd_info_and_img">
   <property name="text">
    <string>下载信息和图片</string>
   </property>
  </action>
  <action name="downlowd_img">
   <property name="text">
    <string>下载图片(需先下载信息)</string>
   </property>
  </action>
  <action name="open_file_action">
   <property name="icon">
    <iconset resource="com/guiTest/pythonQt5/source/resource.qrc">
     <normaloff>:/newPrefix/file.png</normaloff>:/newPrefix/file.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="open_folder_action">
   <property name="icon">
    <iconset resource="com/guiTest/pythonQt5/source/resource.qrc">
     <normaloff>:/newPrefix/folder.png</normaloff>:/newPrefix/folder.png</iconset>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
   <property name="toolTip">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="edit_tab_action">
   <property name="icon">
    <iconset resource="com/guiTest/pythonQt5/source/resource.qrc">
     <normaloff>:/newPrefix/tag.png</normaloff>:/newPrefix/tag.png</iconset>
   </property>
   <property name="text">
    <string>编辑标签</string>
   </property>
   <property name="toolTip">
    <string>编辑标签</string>
   </property>
  </action>
  <action name="downlowd_infoaction_action">
   <property name="icon">
    <iconset resource="com/guiTest/pythonQt5/source/resource.qrc">
     <normaloff>:/newPrefix/download_info.png</normaloff>:/newPrefix/download_info.png</iconset>
   </property>
   <property name="text">
    <string>下载信息</string>
   </property>
   <property name="toolTip">
    <string>下载信息</string>
   </property>
  </action>
  <action name="downlowd_img_action">
   <property name="icon">
    <iconset resource="com/guiTest/pythonQt5/source/resource.qrc">
     <normaloff>:/newPrefix/download_img.png</normaloff>:/newPrefix/download_img.png</iconset>
   </property>
   <property name="text">
    <string>下载图片(需先下载信息)</string>
   </property>
   <property name="toolTip">
    <string>下载图片(需先下载信息)</string>
   </property>
  </action>
  <action name="open_setting_action">
   <property name="icon">
    <iconset resource="com/guiTest/pythonQt5/source/resource.qrc">
     <normaloff>:/newPrefix/settings.png</normaloff>:/newPrefix/settings.png</iconset>
   </property>
   <property name="text">
    <string>打开设置</string>
   </property>
   <property name="toolTip">
    <string>打开设置</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="com/guiTest/pythonQt5/source/resource.qrc"/>
 </resources>
 <connections/>
</ui>
